src/components/WorkspaceEmptyStateSection.tsx

```diff
@@ -24,7 +27,7 @@ type WorkspaceEmptyStateSectionProps = {
     shouldStyleAsCard?: boolean;
 };
 
-function WorkspaceEmptyStateSection({icon, subtitle, title, containerStyle, shouldStyleAsCard = true}: WorkspaceEmptyStateSectionProps) {
+function WorkspaceEmptyStateSection({icon, subtitle, title, containerStyle, shouldStyleAsCard = true, subtitleComponent}: WorkspaceEmptyStateSectionProps) {
```

Comment:
Do we want to default this to be `false`?

This is essentially to give consistency when there is no subtitle?

Reply:
It is a React Node, do we want to use false here? It is checked against undefined later.

Reply:
Nevermind me - I read this wrong. Resolving!